code/datums/movement_detector.dm

```diff
@@ -0,0 +1,55 @@
+/// A datum to handle the busywork of registering signals to handle in depth tracking of a movable
+/datum/movement_detector
+	var/atom/movable/tracked
+	var/datum/callback/listener
```

Comment:
Can you add a `///` comment on listener explaining what the callback is for/when it is invoked

Reply:
done